torchci/lib/drciUtils.ts

```diff
@@ -15,7 +15,7 @@ export const HUD_URL = "https://hud.pytorch.org/pr/";
 
 export function formDrciHeader(prNum: number): string {
     return `## :link: Helpful Links
-### :test_tube: See artifacts and rendered test results [here](${HUD_URL}${prNum})
+### :test_tube: See artifacts and rendered test results [at hud.pytorch.org/pr/${prNum}](${HUD_URL}${prNum})
```

Comment:
nit: the at should not be in the parentheses

Reply:
Also nit: I'm not a big fan of the Note: link to docs will display an error ... 

Is it possible to conditionally show those links?

Reply:
> nit: the at should not be in the parentheses

This is actually a mistake that was leftover from the original implementation of HUD, so would be better to fix probably